node/node_test.go

```diff
@@ -300,11 +297,13 @@ func TestSpacemeshApp_GrpcService(t *testing.T) {
 	r.NoError(err)
 	r.Empty(app.Config.API.PublicServices)
 
-	_, err = grpc.Dial(
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
The reduction in timeout is intentional? maybe the longer timeout is necessary on slower builders?

Reply:
Yes... this test checks if the GRPC service is not running when not enabled. It will basically wait for the whole timeout duration. I think 1 second is more than enough.